docs/en/observability/logs-stream.asciidoc

```diff
@@ -133,20 +133,279 @@ image::images/stream-logs-example.png[example search query on the logs stream pa
+        "_id": "RsWy3IkB8yCtA5VGOKLf",
+        "_score": 1,
+        "_source": {
+          "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.",
+          "@timestamp": "2023-08-09T17:19:27.73312243Z"
+        }
+      }
+    ]
+  }
+}
+----
+
+Notice the text in your message isn't parsed, so you can't filter by any individual fields. Also, the `@timestamp` field shows when you added the data to {es}, not when the log occurred. In this format, neither of these fields are particularly useful for sorting or filtering. Your message, however, contains all of these potential fields:
+
+- *@timestamp* – `2023-08-08T13:45:12.123Z` – Extracting this field lets you sort logs by date and time. This is helpful when you want to view your logs in the order that they occurred or identify when issues happened.
+- *log.level* – `WARN` – Extracting this field lets you filter logs by severity. This is helpful if you want to  focus on high-severity WARN or ERROR-level logs, and reduce noise by filtering out low-severity INFO-level logs.
+- *host.ip* – `192.168.1.101` – Extracting this field lets you filter logs by the host IP addresses. This is helpful if you want to focus on specific hosts that you’re having issues with or if you want to find disparities between hosts.
+- *message* – `Disk usage exceeds 90%.`
+
+NOTE: These fields are part of the {ecs-ref}/ecs-reference.html[Elastic Common Schema (ECS)]. The ECS defines a common set of fields that you can use across Elasticsearch when storing data, including log and metric data.
+
+See these sections information on extracting specific fields:
+
+- <<logs-stream-extract-timestamp>>
+
+[discrete]
+[[logs-stream-extract-timestamp]]
+=== Extract the `@timestamp` field
+
+This section shows you how to extract the `@timestamp` field from the example log:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+----
+
+To extract the timestamp you need to:
+
+- <<logs-stream-ingest-pipeline>>
+- <<logs-stream-simulate-api>>
+- <<logs-stream-index-template>>
+- <<logs-stream-create-data-stream>>
+
+[discrete]
+[[logs-stream-ingest-pipeline]]
+==== Use an ingest pipeline to extract the `@timestamp`
+
+To extract the `@timestamp` field, use an ingest pipeline with a dissect processor and a date processor. Processors are executed in the order they are defined. The example pipeline runs these processors: 
+
+. The {ref}/dissect-processor.html[dissect processor] extracts a structured field from your unstructured log message based on a pattern you set. In the following example, the dissect processor extracts the timestamp to the `timestamp` field.
+. The {ref}/date-processor.html[date processor] then takes the `timestamp` field extracted by the dissect processor and extracts it to a date field. By default, this date field is called `@timestamp`. The date processor can also set the timezone, change the target field, and change the output format of the timestamp.
```

Comment:
I don't think the date processor is required in this case. ES will be able to parse string timestamps. According to https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html, the default format for `date` fields is `strict_date_optional_time||epoch_millis`. According to https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html, `strict_date_optional_time` supports the formats `yyyy-MM-dd'T'HH:mm:ss.SSSZ` or ` yyyy-MM-dd`. As the timestamp in the log file already uses that format, the date processor is not necessary.

However, if the date looks a bit more complex and is non-standard, the date processor is certainly necessary.

I think in this case, I'd remove the date processor but add a link to the date processor docs and explain that it can be useful for parsing dates that are not in ISO or format epoch_millis.